### DISCUSSION TITLE
Monochrome 4:0:0 support

### DISCUSSION THREAD

**Discussion by bardia-az** [fraunhoferhhi/vvenc#95] [Q&A] [answered]
I noticed that vvencapp does not support monochrome (4:0:0) videos as input. Is vvencFFapp similar? Because when I set the parameter of InputChromaFormat to 400, I get a "segmentation fault" error when it reaches the second frame.

**Comment by adamjw24** [comment, marked as answer]
Please check the current head (f7c9b30 or later). We fixed some of the problems concerning 400 encoding. The simple app does not support it, and it will remain experimental for a while, but the full-featured app can be configured appropriately and should work now. Let us know if you run into any bugs.
